stories/text/text.stories.js

```diff
@@ -0,0 +1,172 @@
+export default {
+    title: 'Components/Text',
+    parameters: {
+        description: `The text control is used to display text. It generally contains the text
+         that developers want apps to display (property: text).`,
+        tags: ['f3', 'a11y', 'theme'],
+        components: ['text']
+    }
+};
+
+export const defaultExample = () => `
+    <p class="fd-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+    ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+    aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore
+    eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+    deserunt mollit anim id est laborum.</p>
+`;
+
+defaultExample.storyName = 'Default';
+defaultExample.parameters = {
+    docs: {
+        iframeHeight: 100,
+        storyDescription: 'Default example of the text component'
+    }
+};
+
+export const wrapping = () => `
+    <h3>Whitespace normal</h3>
+    <p class="fd-text" style="white-space: normal;">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+    ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+    aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore
+    eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+    deserunt mollit anim id est laborum.</p>
+    
+    <h3>Whitespace pre-wrap</h3>
+    <p class="fd-text" style="white-space: pre-wrap;">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore
+eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+deserunt mollit anim id est laborum.</p>
+
+    <h3>Whitespace nowrap</h3>
+    <div style="overflow: auto;">
```

Comment:
Please remove all inline styles - uses classes instead https://github.com/SAP/fundamental-styles/blob/master/.storybook/custom/custom.scss

Reply:
If I got your idea correct, you propose to remove all inline styles from examples and replace them inside `custom.scss`.
In this way, all custom, specific, and not reusable styles for unique examples will be stored in one place.
A huge file with a lot of unique styles, and a lot of classes.

But with the current implementation, we have all data for example in one place, independent, easy to modify, or to expand.
And it's already a common practice in these examples. And yes, I don't like inline styles either, but I think the advantages  of this approach is stronger than disadvantage on the inline styles

Reply:
One of the coding style guidelines we have for fundamental-styles is to avoid inline styles whenever possible. We've made an exception for `fd-slider` as that is a special case. If we start doing this here, it's going to creep in all over the place. Please remove them.

Reply:
I found 424 cases of the inline styles in the stories files, and it's definitely not only the `fd-slider`.
If we start collecting all stuff in custom.scss it will become one huge unsupportable file

Reply:
@EvilAlexei, 95% of these 426 cases the inline styling is used to make the documentation more presentable or if we want to showcase an overflow, the other 5% go to adding a background image for Avatar and slider.

Reply:
It sounds like we have a lot cases to fix then 😃  

There are some legitimate uses of inline styles... I just don't think this qualifies. You're showing off instances of a component that don't exist without inline styles - that doesn't seem right. 
Why are we telling consumers to set inline styles? For this wrapping example, shouldn't we have `<span class='fd-text fd-text--wrap'`, where `--wrap` has `white-space: nowrap`?

Reply:
@InnaAtanasova maybe I get something wrong, but in my opinion that inline styles in this component serve the same purpose, only to show examples how users could use it, they are not the part component itself.

Reply:
@jbadan in this way we should have classes for all styles, for example, `fd-text-color-red`. In my opinion, it's a little overkill.

Reply:
I think each approach has its own pros and cons.
I see really strong pros for an isolated approach, where each example has its own styles,
like incapsulation, easy to support and easy to modify.
And cons - inline styles, but in the case of isolated examples, in my opinion, it's not a big deal.

And pros for `custom.scss` - not inline styles.
cons - all styles in one place, it will be difficult to support this file and isolate styles, and in time really difficult.

If I missed something else - I will be glad for the additional info, I'm still learning about this library and approaches here.

Reply:
> @InnaAtanasova maybe I get something wrong, but in my opinion that inline styles in this component serve the same purpose, only to show examples how users could use it, they are not the part component itself.

Imagine you have 2 fundamental buttons in a div in the documentation. 
```
<div>
    <button class="fd-button">Button 1</button>
    <button class="fd-button">Button 2</button>
</div>
```

In this case your 2 buttons will be rendered next to each other with no spacing, which is ugly. In this case we add inline styling on the container div and you get something like:
```
<div style="display:flex; justify-content: space-between;">
    <button class="fd-button">Button 1</button>
    <button class="fd-button">Button 2</button>
</div>
```

This type of inline styling is ok because it only concerns the docs part and does not affect the components (the user doesn't have to do anything). Your inline styling IS affecting the component directly. For a specific case the user will have to write `<p class="fd-text" style="white-space: nowrap;">` or `<p class="fd-text" style="white-space: pre-wrap;">` or `<p class="fd-text" style="white-space: normal;">`. That's exactly why we have modifier classes in the library. We have 1 default behaviour and we use the modifier classes to overwrite it. 
Also, if you take a look from the implementation library perspective it will be much easier to manipulate this with classes and not with inline styling. The user can pass to the input property the desired value for white-space and internally we will add the required class. 
Anyways, inline styling on the component level is not something we want to have in the library, so you need to change it to modifier classes.

Reply:
@InnaAtanasova @jbadan My bad, I found the documentation with a description about classes for text-component that should have such functionality, I will add them in next commit, and I will try to remove all inline styles